tabs/src/components/sample/TableOfContents.jsx

```diff
@@ -16,32 +22,44 @@ const Headings = ({ headings }) => {
 
   const { t } = useTranslation();
 
+  const checkIfActive = (menuItemKey, activeHeaderId) => {
+    if (menuItemKey === activeHeaderId) {
+      return true;
+    } else {
+      return false;
+    }
+  };
+
```

Comment:
refactor to `return menuItemKey === activeHeaderId`

Reply:
d05ba349ea0351e4fde8d6bbff51dcb0523d4e16